packages/table/src/SelectionBar.js

```diff
@@ -1,25 +1,15 @@
 import * as React from "react";
 import PropTypes from "prop-types";
 import Text, { fontSizes } from "@crave/farmblocks-text";
-import Button, { buttonTypes } from "@crave/farmblocks-button";
+import Button from "@crave/farmblocks-button";
 import StyledBar from "./styledComponents/SelectionBar";
 
 const SelectionBar = props => {
-  const {
-    selectedRows,
-    clearSelection,
-    title,
-    primaryAction,
-    secondaryAction
-  } = props;
-  const hasActions = !!secondaryAction || !!primaryAction;
-  const hasSecondaryButton = !!secondaryAction;
+  const { selectedRows, clearSelection, height, title, children } = props;
```

Comment:
probably ok, but do we really want to reuse the children prop for this? why use a property name that is normally a set of nodes to be a render function? will this break React.Children utilities (not that we use them, just curiosity)?

Reply:
Google searching this pattern of function as child, there are results for and against it, but the main argument of [this one here, against it](https://americanexpress.io/faccs-are-an-antipattern/) is that by using this pattern we lose the ability of having meaningful names, and that we are re-purposing a familiar prop for the sake of clever hackism. I tend to agree with that conclusions.